tasks/components/cards/WardCard.tsx

```diff
@@ -42,18 +50,22 @@ export const WardCard = ({
   onEditClick = () => undefined
 }: PropsWithLanguage<WardCardTranslation, WardCardProps>) => {
   const translation = useTranslation(language, defaultWardCardTranslations)
+  const numberOfBeds = ward.rooms.map(value => value.bedCount).reduce((previousValue, currentValue) => currentValue + previousValue, 0)
+  const hasRooms = ward.rooms.length > 0
+
   return (
     <Card onTileClick={onTileClick} isSelected={isSelected} className={tw('group cursor-pointer')}>
       <div className={tw('flex flex-row justify-between w-full')}>
         <span className={tw('font-bold font-space')}>{ward.name}</span>
         <button onClick={onEditClick}
                 className={tw('hidden group-hover:block')}>{translation.edit}</button>
       </div>
-      <div className={tw('text-left my-1')}>{ward.roomNames.join(', ')}</div>
+      <div
+        className={tx('text-left my-1', { 'text-gray-400': !hasRooms })}>{hasRooms ? ward.rooms.map(value => value.name).join(', ') : translation.noRoomsYet}</div>
```

Comment:
this is a bit too distracting for my taste
maybe make it a little bit smaller and bleach the color a bit